packages/lib/src/typography/index.js

```diff
@@ -0,0 +1,64 @@
+import {
```

Comment:
I believe this is a direct port of functions in govuk-frontend. Can you please include an inline comment with a link to the file(s) this is porting/tracking?